feature_2054_helpEmailChanges (#2064)
dtcenter/MET: met/docs/Users_Guide/appendixA.rst

		
		  MET_BUFRLIB=/home/username/BUFRLIB_v10.2.3 

> After doing that, please try recompiling MET. If it fails, please submit the following log files: "make_install.log" as well as "config.log".them to `METplus GitHub Discussions Forum <https://github.com/dtcenter/METplus/discussions>`_

Reviewer: ```suggestion
After doing that, please try recompiling MET. If it fails, please submit the "make_install.log" file as well as the "config.log" file with a new post in the `METplus GitHub Discussions Forum <https://github.com/dtcenter/METplus/discussions>`_.
```
